src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -91,27 +131,27 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
         })
         .search();
     };
-    const getShowMore = helper => () => {
+    const getShowMore = (helper): (() => void) => () => {
       helper.setPage(lastReceivedPage + 1).search();
     };
 
     return {
       getConfiguration() {
-        return escapeHTML ? TAG_PLACEHOLDER : undefined;
+        return escapeHTML ? TAG_PLACEHOLDER : {};
       },
 
       init({ instantSearchInstance, helper }) {
-        this.showPrevious = getShowPrevious(helper);
-        this.showMore = getShowMore(helper);
-        firstReceivedPage = helper.state.page;
-        lastReceivedPage = helper.state.page;
+        (this as any).showPrevious = getShowPrevious(helper);
+        (this as any).showMore = getShowMore(helper);
+        firstReceivedPage = helper.state.page as number;
```

Comment:
According to the `algoliasearch-helper` type definition `page` can be `undefined` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/algoliasearch-helper/index.d.ts#L429

Is this true or is this an error in the type definition? Because it is a case that is not handled by the widget at this time (this is why we force to `number` type here)

Reply:
You can use `helper.state.page!`. I don't think it can be `undefined`. If needed, you can override the type in:
https://github.com/algolia/instantsearch.js/blob/9741445905265ffb0f3e770fbea9c8e0b0ac4849/src/types/instantsearch.ts#L110